### PR TITLE
Adds getting started installation instructions

### PR DESCRIPTION
**Pull request by ConorOkus**
[Preview](https://deploy-preview-133--lightningdevkit.netlify.app/tutorials/getting-started/)

### CODE REVIEW

File: docs/tutorials/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## System Requirements
+MacOS, Windows and Linux are supported.
+
+## Installation
+To add LDK to a project, run:
+
+:::: tabs
+::: tab "Rust"
+```rust
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:15]: Does the markdown used for the site supports toml syntax highlighting? If not may want to drop the language here.

File: docs/tutorials/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+/*
+Add the following dependencies to your cargo.toml and replace {VERSION} with the 
+version number you want to use from
+*/
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:19]: Toml only supports comments prefixed with `#`.

File: docs/tutorials/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+::: tab "Javascript/Typescript"
+```javascript
+npm i lightningdevkit --save
+```
+:::
+::::
+
+Example usage after installation is complete:
+:::: tabs
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:81]: Would be great if the language tabs here and the previous box were synced. I think that had been done in one of the guides when we had more than one example per page prior to breaking it up into subpages.

File: docs/tutorials/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+::: tab "Rust"
+```rust
+use lightning::chain::chaininterface::{FeeEstimator};
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:84]: No need for braces when there's only on type imported.

File: docs/tutorials/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+MacOS, Windows and Linux are supported.
+
+## Installation
+To add LDK to a project, run:
+
+:::: tabs
+::: tab "Rust"
+```rust
+/*
+Add the following dependencies to your cargo.toml and replace {VERSION} with the
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:17]: No braces on `VERSION` below.

File: docs/tutorials/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+## Installation
+To add LDK to a project, run:
+
+:::: tabs
+::: tab "Rust"
+```rust
+/*
+Add the following dependencies to your cargo.toml and replace {VERSION} with the 
+version number you want to use from
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:18]: Little cleaner if you drop "from". Likewise elsewhere.

File: docs/tutorials/getting-started.md (hunk 1 above)
Comment (ConorOkus) [reply docs/tutorials/getting-started.md]: Updated in https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/14f55d2f075ea521b1097ddb1ce19af403df30bc

File: docs/tutorials/getting-started.md (hunk 2 above)
Comment (ConorOkus) [reply docs/tutorials/getting-started.md]: Updated in https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/14f55d2f075ea521b1097ddb1ce19af403df30bc

File: docs/tutorials/getting-started.md (hunk 4 above)
Comment (ConorOkus) [reply docs/tutorials/getting-started.md]: Fixed [03764a8](https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/03764a8334a6940e905a41d80753400f9e659e71)

File: docs/tutorials/getting-started.md (hunk 6 above)
Comment (ConorOkus) [reply docs/tutorials/getting-started.md]: Fixed [03764a8](https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/03764a8334a6940e905a41d80753400f9e659e71)

File: docs/tutorials/getting-started.md (hunk 3 above)
Comment (ConorOkus) [reply docs/tutorials/getting-started.md]: cc @dennisreimann seems like it's possible on page load but not when you actually click a single tab?

Comment (jkczyz) [reply docs/tutorials/getting-started.md]: Ran across this which may help? https://github.com/vuejs/vuepress/issues/851#issuecomment-876438287

Comment (dennisreimann) [reply docs/tutorials/getting-started.md]: @jkczyz Thanks for the pointer, I've migrated to the code switcher plugin in #139.

**Comment by ConorOkus** [comment]
Thanks for the CodeSwitcher recommendation @jkczyz. Updated in https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/b1d8e3163b5d412bff5fd30d9d10defdd4b6625a

File: docs/tutorials/getting-started.md [hunk 7]
```diff
@@ -6,34 +6,34 @@ If you have any questions about anything related to LDK, feel free to ask our co
...
-lightning-background-processor = { version = {VERSION} }
-lightning-rapid-gossip-sync = { version = {VERSION} }
-```
-:::
-
-::: tab Java
+<CodeSwitcher :languages="{rust:'Rust', java:'Java', kotlin:'Kotlin', javascript:'JavaScript'}">
+  <template v-slot:rust>
+ 
+  ```toml
```
Comment (jkczyz) [inline docs/tutorials/getting-started.md:15]: Indentation levels seem inconsistent across the first `CodeSwitcher` block, FWIW. The second block seems fine though with two spaces and one within the formatted language blocks.

**Pull request merged**